Gems/Compression/Code/Source/CompressionModuleInterface.cpp

```diff
@@ -9,9 +9,12 @@
 #include "CompressionModuleInterface.h"
 #include <AzCore/Memory/Memory.h>
 
+#include <Clients/CompressionSystemComponent.h>
+
 #include <Compression/CompressionTypeIds.h>
+#include <Compression/DecompressionInterfaceAPI.h>
 
-#include <Clients/CompressionSystemComponent.h>
+#include <Clients/DecompressionRegistrarImpl.h>
```

Comment:
nit: Can we group this include together with the `#include <Clients/CompressionSystemComponent.h>` above?

Reply:
Yes we can